static/style.css

```diff
@@ -106,30 +106,44 @@ button:focus {
     margin-bottom: 40px;
 }
 
+
+
 .button {
-    margin: 0 5px;
-    padding: 10px;
-    border: 2px solid black;
-    border-radius: 4px;
-    text-decoration: none;
-    color: black;
-    font-weight: bold;
+    font-size:1em;
```

Comment:
Nitpick: please add spaces after the : for consistency with the rest of the file.